docs/evaluate/use-cases-design-patterns.mdx

```diff
@@ -0,0 +1,111 @@
+---
+id: use-cases-design-patterns
+title: Temporal Use Cases and Design Patterns 
+description: Discover various use cases and architectural design patterns for implementing Workflows with Temporal.
+sidebar_label: Use Cases
+tags:
+  - temporal-use-cases
+  - temporal-design-patterns
+keywords:
+  - temporal use cases
+  - temporal design patterns
+  - temporal workflows
+---
+
+This page provides an overview of how leading organizations leverage Temporal to solve real-world problems, general use cases, and architectural design patterns. 
+
+## Use Cases of Temporal in Production
+
+Here are some examples where Temporal is most impactful and running in production at large organizations today.
+
+### Transactions
+
+Actions or activities involving two or more parties or things that reciprocally affect or influence each other. For example:
+
+- Payment processing at [Stripe](https://www.youtube.com/watch?v=Crkcr1S-NSc)
```

Comment:
In general it's best not to put "raw" URLs like this; we want to be able to see how traffic is flowing on the website. Let's go instead for:

```suggestion
- Payment processing at [Stripe](https://temporal.io/resources/on-demand/stripe)
```